src/Workspaces/CSharp/Portable/SemanticModelReuse/CSharpSemanticModelReuseLanguageService.cs

```diff
@@ -31,11 +32,15 @@ public CSharpSemanticModelReuseLanguageService()
 
         protected override ISyntaxFacts SyntaxFacts => CSharpSyntaxFacts.Instance;
 
-        protected override SyntaxList<AccessorDeclarationSyntax> GetAccessors(EventDeclarationSyntax @event)
-            => @event.AccessorList?.Accessors ?? default;
+        protected override MemberDeclarationSyntax GetAccessorContainerDeclaration(AccessorDeclarationSyntax currentAccessor)
+        {
+            Contract.ThrowIfFalse(currentAccessor.Parent is AccessorListSyntax);
+            Contract.ThrowIfFalse(currentAccessor.Parent.Parent is MemberDeclarationSyntax);
+            return (MemberDeclarationSyntax)currentAccessor.Parent.Parent;
+        }
 
-        protected override SyntaxList<AccessorDeclarationSyntax> GetAccessors(PropertyDeclarationSyntax property)
-            => property.AccessorList?.Accessors ?? default;
+        protected override SyntaxList<AccessorDeclarationSyntax> GetAccessors(MemberDeclarationSyntax member)
```

Comment:
❔ Why not `BasePropertyDeclarationSyntax`?

Reply:
taht def works :)